### PR TITLE
Adding bootstrap to project

### PR DESCRIPTION
**Pull request by RyPope**
Allows user to customize their layouts much easier.

Added config to ignore and replaced with example config (so people don't
accidently commit it).

### CODE REVIEW

**Comment by RyPope** [comment]
Sorry to the merge spam. Accidentally did the exact thing I was trying to accomplish in this commit and committed my config like a moron.

File: web/includes/generatetopplayers.php [hunk 1]
```diff
@@ -1,24 +1,23 @@
...
-if (isset($_GET['serverid'])) {
-	$server_id = (int)$_GET['serverid'];
-	$server_limit = 'AND serverID='.(int)$_GET['serverid'];
-}
+$server_limit = '';
+if (isset($_GET['server_id']))
+	$server_limit = 'AND serverID='.(int)$_GET['server_id'];
 
-$run_query = "SELECT s1.*, (SELECT COUNT(*) FROM $mysql_table AS s2 WHERE s2.rating > s1.rating AND s2.wins+s2.losses > 200 $server_limit)+1 AS rank FROM $mysql_table AS s1 WHERE s1.wins+s1.losses > 200 $server_limit ORDER BY rating DESC LIMIT 0, 15";
+$run_query = "SELECT s1.*, (SELECT COUNT(*) FROM $mysql_table AS s2 WHERE s2.rating > s1.rating AND s2.wins+s2.losses > 100)+1 AS rank FROM $mysql_table AS s1 WHERE s1.wins+s1.losses > 100 $server_limit ORDER BY rating DESC LIMIT 0, 15";
```
Comment (mattikus) [inline web/includes/generatetopplayers.php]: These constants should probably become config vars.

**Comment by mattikus** [comment]
Why pull in all of bootstrap in the development versions? Seems like a large thing to include in this instead of making it part of the installation step.

Also, this should honestly probably be forked off into it's own project and separated from the core 1v1 repository.

**Comment by RyPope** [comment]
I can use a CDN instead I suppose but no need to complicate installation. Bootstrap + jQuery is not that big anyway.

File: web/includes/generatetopplayers.php (hunk 1 above)
Comment (RyPope) [inline web/includes/generatetopplayers.php]: Agreed

**Comment by mattikus** [comment]
Well, you should definitely fix the `readme` to note your change to the naming for the config.inc.php. Probably also update the contact information.

**Comment by RyPope** [comment]
Sounds good. Contact information?

**Comment by mattikus** [comment]
in the [other info](web/readme.md#other-info) section of the readme, the previous author added a note about where to contact for assistance.

**Comment by splewis** [comment]
I think given the relative complexity and number of dependencies this adds (and my lack of knowledge in all things web-dev related), I think it'd be better to have this in a separate repository. (I actually regret having the simplified one in this repository)

I'd be happy to include links to other implementations of stats website, etc.

**Pull request closed without merging**